script/codecovflags

```diff
@@ -0,0 +1,1550 @@
+#!/usr/bin/env bash
```

Comment:
I just saw the [codecov](https://github.com/codecov/codecov-python) pip package page. The installed package supports flags. We don't need a dedicated bash script for the same.

We are already installing the codecov pip package for all the test types. You can reuse the same using the command
```
> codecov -F <test_type>
```

You can delete this script.